core/templates/pages/Base.ts

```diff
@@ -108,8 +108,8 @@ angular.module('oppia').controller('Base', [
         {
           propertyType: 'property',
           propertyValue: 'og:image',
-          content: $scope.getAssetUrl(
-            '/assets/images/logo/288x288_logo_mint.png')
+          content: $scope.getStaticImageUrl(
+            '/logo/288x288_logo_mint.webp')
```

Comment:
Could we support both PNG and WebP here?

Reply:
I have made them as picture tags in the directive.html file

Reply:
No, what is the directive file for `Base.ts`, also why do we need this variable if you set it up in the HTML file.